torch/utils/checkpoint.py

```diff
@@ -226,7 +226,7 @@ def checkpoint(function, *args, use_reentrant: bool = True, **kwargs):
         preserve_rng_state(bool, optional):  Omit stashing and restoring
             the RNG state during each checkpoint.
             Default: ``True``
-        use_reentrant(bool, optional): Use checkpointing
+        use_reentrant(bool, optional): Use (the default) checkpointing
```

Comment:
Not sure what that means? There is a `Default: True` in this section.

Reply:
I think this is already done by the `Default:` specification (note that we (try to) use this form consistently across the doc. So the user expects to find that information there already.

Reply:
sg, will remove